game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -333,11 +333,21 @@ public void updateTiledMapRenderer()
         this.currentNPCS = game.getNPCS(game.player.getRoom());
     }
 
+    public boolean getChangeMap()
+    {
+        return(changeMap);
```

Comment:
Should I just get rid of the NavigationScreen tests then? The "getNPC" getter was one that was already there, I didn't add it, I'll just test for it then.